gwctl/cmd/describe.go

```diff
@@ -45,6 +47,7 @@ func NewDescribeCommand() *cobra.Command {
 	}
 	cmd.Flags().StringVarP(&namespaceFlag, "namespace", "n", "default", "")
 	cmd.Flags().BoolVarP(&allNamespacesFlag, "all-namespaces", "A", false, "If present, list requested resources from all namespaces.")
+	cmd.Flags().StringVarP(&labelSelector, "selector", "l", "", "Label selector.")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&labelSelector, "selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2). Matching objects must satisfy all of the specified label constraints.")
```
minor nit 

Inspired from here - https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/util/helpers.go#L524 (something which has been a part of k8s ecosystem since a while :) )